tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -410,40 +413,46 @@ SPECTRE_TEST_CASE("Unit.DataStructures.DataVector.Math",
   const DataVector e11{-20.0, -20.0, -20.0, -20.0};
   const DataVector e12{-30.0, -30.0, -30.0, -30.0};
 
-  const std::array<double, 3> point{{10.0, 20.0, 30.0}};
```

Comment:
I don't like scoping (personally, but that's not all too important). The difficulty is that we have to check every single like very carefully, ideally with a script to make sure they are the same. This buts a lot of burden on the reviewers. If there was something missing, it would be useful to add.